cid/helpers/cur_proxy.py

```diff
@@ -0,0 +1,452 @@
+    'product_max_iopsvolume': "product['max_iopsvolume']",
+    'product_max_throughputvolume': "product['max_throughputvolume']",
+    'product_max_volume_size': "product['max_volume_size']",
+    'product_memory': "product['memory']",
+    'product_message_delivery_frequency': "product['message_delivery_frequency']",
+    'product_message_delivery_order': "product['message_delivery_order']",
+    'product_min_volume_size': "product['min_volume_size']",
+    'product_network_performance': "product['network_performance']",
+    'product_normalization_size_factor': "product['normalization_size_factor']",
+    'product_operating_system': "product['operating_system']",
+    'product_physical_processor': "product['physical_processor']",
+    'product_platopricingtype': "product['platopricingtype']",
+    'product_platovolumetype': "product['platovolumetype']",
+    'product_pre_installed_sw': "product['pre_installed_sw']",
+    'product_processor_architecture': "product['processor_architecture']",
+    'product_processor_features': "product['processor_features']",
+    'product_queue_type': "product['queue_type']",
+    'product_region': "product['region']",
+    'product_request_type': "product['request_type']",
+    'product_servicename': "product['servicename']",
+    'product_storage': "product['storage']",
+    'product_storage_class': "product['storage_class']",
+    'product_storage_media': "product['storage_media']",
+    'product_tenancy': "product['tenancy']",
+    'product_transfer_type': "product['transfer_type']",
+    'product_vcpu': "product['vcpu']",
+    'product_version': "product['version']",
+    'product_volume_api_name': "product['volume_api_name']",
+    'product_volume_type': "product['volume_type']",
+    'product_vpcnetworkingsupport': "product['vpcnetworkingsupport']",
+    'product_edition': "product['edition']",
+    'product_gpu_memory': "product['gpu_memory']",
+    'product_pack_size': "product['pack_size']",
+    'product_q_present': "product['q_present']",
+    'product_subscription_type': "product['subscription_type']",
+    'product_usage_group': "product['usage_group']",
+    'product_cache_engine': "product['cache_engine']",
+    'product_invocation': "product['invocation']",
+    'product_memory_gib': "product['memory_gib']",
+    'product_time_window': "product['time_window']",
+    'product_finding_group': "product['finding_group']",
+    'product_finding_source': "product['finding_source']",
+    'product_finding_storage': "product['finding_storage']",
+    'product_standard_group': "product['standard_group']",
+    'product_standard_storage': "product['standard_storage']",
+    'product_product_type': "product['product_type']",
+    'product_pricingplan': "product['pricingplan']",
+    'product_provider': "product['provider']",
+    'product_subservice': "product['subservice']",
+    'product_type': "product['type']",
+    'product_tickettype': 'n/a',
```

Comment:
Why this fields are mapped to n/a?  What makes you think they don't exist in CUR 2.0?

Reply:
In my account I have cur1 and cur2 so I pulled all fields of both and those were missing.

Reply:
I think it is because you for example don't have resources with extended support for this column for example product_extended_support_pricing_year.  In reality I think they are all mapped 1:1 to old fields.